OpenRA.Game/Network/Nat.cs

```diff
@@ -0,0 +1,108 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2021 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Threading;
```

Comment:
Remove

Reply:
SemaphoreSlim needs it.

Reply:
Strange, resharper reported this as unnecessary.

Reply:
VS Code doesn't.

Reply:
it was the `using System.Threading.Tasks;` that was clearly removed earlier.